ddtrace/contrib/tornado/handlers.py

```diff
@@ -57,6 +57,14 @@ def execute(func, handler, args, kwargs):
 
         setattr(handler.request, REQUEST_SPAN_KEY, request_span)
 
+        for rule in handler.application.wildcard_router.rules:
```

Comment:
On a very superficial look at tornado and saw there's also the `default_router` which, from what I understood, may contain different routes. Should we look into that as well?

Reply:
I went through and updated the code to handle both the default router and hierarchical rules